al-khaser/Anti Debug/UnhandledExceptionFilter_Handler.cpp

```diff
@@ -19,7 +19,8 @@ LONG WINAPI UnhandledExcepFilter(PEXCEPTION_POINTERS pExcepPointers)
 
 BOOL UnhandledExcepFilterTest ()
 {
-	SetUnhandledExceptionFilter(UnhandledExcepFilter);
-	RaiseException(EXCEPTION_FLT_DIVIDE_BY_ZERO, 0, 0, NULL); 
+	LPTOP_LEVEL_EXCEPTION_FILTER Top = SetUnhandledExceptionFilter(UnhandledExcepFilter);
+	RaiseException(EXCEPTION_FLT_DIVIDE_BY_ZERO, 0, 0, NULL);
+	SetUnhandledExceptionFilter(Top);

```

Comment:
This is to restore the previous exception filter established with the function ?